src/CoreWCF.Templates/src/templates/CoreWCFService/.template.config/template.json

```diff
@@ -118,8 +122,8 @@
           "description": "Target net462"
         }
       ],
-      "replaces": "net6.0",
-      "defaultValue": "net6.0"
+      "replaces": "net8.0",
```

Comment:
I don't think the default should be net8.0 until after net8.0 is released GA

Reply:
@mconnew i restored default to be net6.0